apply-cluster.sh

```diff
@@ -26,6 +30,54 @@ kubectl create configmap prometheus-cluster-config \
     --from-file=config/cluster/prometheus \
     --dry-run="client" -o json | kubectl apply -f -
 
+# Construct the per-project CLIENT_ID, CLIENT_SECRET and COOKIE_SECRET for OAuth.
+export OAUTH_PROXY_CLIENT_ID=OAUTH_PROXY_CLIENT_ID_${PROJECT/-/_}
+export OAUTH_PROXY_CLIENT_SECRET=OAUTH_PROXY_CLIENT_SECRET_${PROJECT/-/_}
+export OAUTH_PROXY_COOKIE_SECRET=OAUTH_PROXY_COOKIE_SECRET_${PROJECT/-/_}
+
+# Replace template variables in oauth2-proxy.yml.
+sed -i -e 's|{{OAUTH_PROXY_CLIENT_ID}}|'${!OAUTH_PROXY_CLIENT_ID}'|g' \
+    -e 's|{{OAUTH_PROXY_CLIENT_SECRET}}|'${!OAUTH_PROXY_CLIENT_SECRET}'|g' \
+    -e 's|{{OAUTH_PROXY_COOKIE_SECRET}}|'${!OAUTH_PROXY_COOKIE_SECRET}'|g' \
+    k8s/data-pipeline/deployments/oauth2-proxy.yml
+
+# Additional k8s resources installed via Helm
+#
+# Download Helm
+curl -O https://get.helm.sh/helm-${K8S_HELM_VERSION}-linux-amd64.tar.gz
+tar -zxvf helm-${K8S_HELM_VERSION}-linux-amd64.tar.gz
+
+# Add repos
+./linux-amd64/helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+#./linux-amd64/helm repo add jetstack https://charts.jetstack.io
```

Comment:
Does this jetstack repo need to be uncommented for the cert-manager install to work?

Reply:
Yes. (Moved to `cloudbuild.yaml`)